src/foundry/client/pixi/placeables/primary-canvas-objects/primary-sprite-mesh.d.mts

```diff
@@ -0,0 +1,110 @@
+import type { InexactPartial } from "../../../../../types/utils.d.mts";
+
+// TODO: Remove when #2570 is completed
+declare const PrimaryBaseSamplerShader: BaseSamplerShader;
+
+// TODO: Remove when #2878 is completed
+type TextureAlphaData = unknown;
```

Comment:
Could you also note these TODOs on these issues?

Reply:
Already done